### PR TITLE
Restructuring so we can include new clouds and new languages in our workshops

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,45 +6,32 @@ This workshop uses AWS although most concepts are applicable no matter what clou
...
 * [Continuous delivery](https://www.pulumi.com/docs/guides/continuous-delivery/)
 * [Secrets management](https://www.pulumi.com/blog/managing-secrets-with-pulumi/)
 * [Multi-project infrastructure architectures](https://www.pulumi.com/blog/architect-aws-application-infra-with-pulumi-stack-references/)
 * [Test-driven infrastructure](https://www.pulumi.com/blog/unit-testing-infrastructure-in-nodejs-and-mocha/)
 
-Thank you for checking out the Infrastructure as Code Workshop! More labs are on their way. Please [file an issue](https://github.com/joeduffy/infrastructure-as-code-workshop/issues/new) if there are topics you'd like to see covered in the future.
+Thank you for checking out the Infrastructure as Code Workshop! More labs are on their way. Please [file an issue](https://github.com/pulumi/infrastructure-as-code-workshop/issues/new) if there are topics you'd like to see covered in the future.
+
+[1]: ./aws/typescript/README.md
+[2]: ./aws/python/README.md
```
Comment (mikhailshilkov) [inline README.md:37]: these links seem broken: missing the "labs" part

File: README.md [hunk 2]
```diff
@@ -6,45 +6,32 @@ This workshop uses AWS although most concepts are applicable no matter what clou
...
-The second lab takes you on a tour of cloud architectures, including VMs, containers, and serverless:
+A course can be language and cloud specific. Choose the link to the specific course you feel is applicable and you can get started
 
-1. [Provisioning EC2 Virtual Machines](./labs/02-app-arch/01-provisioning-vms.md)
-3. [Deploying Containers to Elastic Container Service (ECS) "Fargate"](./labs/02-app-arch/02-containers-on-ecs.md)
-4. [Deploying Containers to a Kubernetes Cluster](./labs/02-app-arch/03-containers-on-kubernetes.md)
-5. [Using AWS Lambda for Serverless Application Patterns](./labs/02-app-arch/04-lambda-serverless.md)
+|            | TypeScript            | Python                   | C#                       | 
+| ------     | -------               | -------                  | -------                  |
+| AWS        |   [get started][1]    |   [get started][2]       |   Coming soon            |
```
Comment (mikhailshilkov) [inline README.md:21]:
```suggestion
| AWS        |   [Get started][1]    |   [Get started][2]       |   Coming soon            |
```

**Review by mikhailshilkov — APPROVED** [review]
I haven't checked the content apart from top-level readme, but the structure makes sense to me and we can make further improvements in separate PRs. Ship it, and I'll add Azure/C#.

**Comment by stack72** [comment]
Merging and we can start to improve the navigation

**Pull request merged**